Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.cpp

```diff
@@ -235,10 +235,14 @@ void NetworkRTCProvider::stopResolver(LibWebRTCResolverIdentifier identifier)
 #if PLATFORM(COCOA)
 const String& NetworkRTCProvider::attributedBundleIdentifierFromPageIdentifier(WebPageProxyIdentifier pageIdentifier)
 {
-    return m_attributedBundleIdentifiers.ensure(pageIdentifier, [&]() -> String {
+    return m_attributedBundleIdentifiers.ensure(pageIdentifier, [checkedThis = CheckedPtr { this }, pageIdentifier]() -> String {
```

Comment:
Seems we should create a `protectedThis` right away here instead of a `CheckedThis` since you end up synchronously constructing a protectedThis` a few lines below anyway.